src/utils/types/connector-types.ts

```diff
@@ -0,0 +1,73 @@
+import { BigNumber } from "ethers";
+import { Chain } from "./networks";
+
+export interface Pool {
+  name: string;
+  chain: Chain | string;
+  underlying_tokens: string[];
+  pool_address: string;
+  investing_address: string;
+  staking_address: string | null;
+  boosting_address: string | null;
+  distributor_address: string | null;
+  rewards_tokens: string[] | null;
+  metadata: object;
+}
+
+export interface Amount {
+  humanValue: string;
```

Comment:
wdyt of `amountWithoutDecimals` and `amountWIthDecimals` ?

Reply:
without decimals being the on-chain one and with decimals being the human readable one ?
We type one to BigNumber and the other one as a number ?

Reply:
Yep exactly, wdyt of that